tools/reset_github_user_token_and_update_circleci.sh

```diff
@@ -64,35 +71,58 @@ fi
 # GitHub: create a user-scope token
 # TODO: Delete any existing token for that repo
 echo "Github: Creating a Github user-scoped token for each pack"
-echo
 
+NUM_PACKS=$#
+
+PACK_NUM=0
 for PACK in $@; do
+    PACK_NUM=$(($PACK_NUM+1))
     REPO_NAME=$(repo_name ${PACK})
-    echo "REPO_NAME=${REPO_NAME}"
+    SCOPE=$(repo_scope ${PACK})
+
+    # If we're doing more than one pack, add some whitespace and a counter
+    if [[ $NUM_PACKS -gt 1 ]]; then
+        # Space things out a bit
+        echo
+        echo
+
+        # Use printf instead of echo since it doesn't include a newline terminator
+        printf "[${PACK_NUM}/${NUM_PACKS}]"
```

Comment:
```suggestion
        # Space things out a bit
        echo
        echo

        # Use printf instead of echo since it doesn't include a newline terminator
        printf "[${PACK_NUM}/${NUM_PACKS}]"
```

is there a better construction than this?

Reply:
I don't understand your suggestion. Could you edit/rephrase?

Reply:
This script doesn't need to be perfect to be useful.

Reply:
At least it's possible to do
```bash
printf "\n\n[${PACK_NUM}/${NUM_PACKS}]"
```

Reply:
Fixed